src/containerConfig.ts

```diff
@@ -23,7 +24,16 @@ export const registerExternalValues = async (options?: RegisterOptions): Promise
 
   try {
     const loggerConfig = config.get<LoggerOptions>('telemetry.logger');
-    const logger = jsLogger({ ...loggerConfig, mixin: getOtelMixin() });
+    const logger = jsLogger({
+      ...loggerConfig,
+      mixin: getOtelMixin(),
+      redact: {
+        paths: ['args.H', 'args.dumpServerHeaders', 'args["dump-server-headers"]'],
+        censor: (headersKeyValArr: string[]) => {
+          return Object.keys(parseHeadersArg(headersKeyValArr));
+        },
```

Comment:
just to be clear, you are redacting all the headers instead of the ones that should be redacted like x-api-key?
sounds like a dictatorship to me.